Microsoft.Identity.Web/ClaimPrincipalExtension.cs

```diff
@@ -107,10 +54,10 @@ public static string GetDomainHint(this ClaimsPrincipal claimsPrincipal)
         /// <remarks>See https://docs.microsoft.com/en-us/azure/active-directory/develop/id-tokens#payload-claims </remarks>
         public static string GetDisplayName(this ClaimsPrincipal claimsPrincipal)
         {
-            // Attempting the claims brought by an Microsoft identity platform token first
+            // Use the claims in an Microsoft identity platform token first
```

Comment:
Already fixed in the Microsoft.Identity.Web master source (in the ASP.NET Core Web app sample)